internal/host/statemachine.go

```diff
@@ -385,6 +386,34 @@ func NewHostStateMachine(sm stateswitch.StateMachine, th *transitionHandler) sta
 		PostTransition:   th.PostRefreshHost(statusInfoDisconnected),
 	})
 
+	sm.AddTransition(stateswitch.TransitionRule{
+		TransitionType: TransitionTypeMediaDisconnect,
+		SourceStates: []stateswitch.State{
+			stateswitch.State(models.HostStatusDiscovering),
+			stateswitch.State(models.HostStatusInsufficient),
+			stateswitch.State(models.HostStatusKnown),
+			stateswitch.State(models.HostStatusPendingForInput),
+			stateswitch.State(models.HostStatusDisconnected),
+			stateswitch.State(models.HostStatusBinding),
+		},
+		DestinationState: stateswitch.State(models.HostStatusDisconnected),
```

Comment:
The disconnected state has been used so far to indicate a host that stopped communicating with the service. I think overloading it to also mean "Media Disconnected" is a bit confusing

Reply:
I had a discussion with @avishayt and @romfreiman about that a month ago and we decided that disconnection could be network or media disconnection.

Reply:
please read my next comment